app/components/concept.ts

```diff
@@ -150,14 +184,21 @@ export default class ConceptComponent extends Component<Signature> {
       });
 
       this.updateLastRevealedBlockGroupIndex(this.currentBlockGroupIndex - 1);
+      this.enqueueConceptEngagementUpdate.perform();
     }
 
     // TODO: Add analytics event?
   }
 
+  @action
+  handleWillDestroyContainer() {
+    if (!this.authenticator.isAuthenticated && config.environment !== 'test') {
```

Comment:
@libmartinito what was the deal with this in the test environment? what kind of test failure do we see if this test environment filter is removed?

Reply:
tried using unloadRecord. didn't work. will make an issue after this gets merged so i could reference code. will update here after making issue